### ISSUE TITLE
SetUserId applies only to crashes?

### ISSUE THREAD

**Issue by breyed** [Microsoft/AppCenter-SDK-DotNet#828]
It's not clear what scope `AppCenter.SetUserId` applies to. It is documented anywhere? When I set it, I don't see any user IDs in my events exported to Application Insights. [A post today](https://blogs.msdn.microsoft.com/vsappcenter/introducing-diagnostics-search-and-user-id-support/) mentions user IDs in the context of crashes, but not events.

It would be useful for the user ID to be incorporated into both AppCenter's dashboard for events and its export of events to Application Insights.

Version 1.13.0

**Comment by blparr** [comment]
Hello, thanks for your feedback!

User ID can only be applied to Diagnostics (crashes, errors) for now. 

We will probably be extending support for events, sessions and other analytics data, but there is no ETA for it just yet. We will tag your request for future consideration. Thanks for bringing this up.

**Comment by dhei** [comment]
@breyed the `AppCenter.SetUserId` API is documented at [Identify users](https://docs.microsoft.com/en-us/appcenter/sdk/other-apis/xamarin#identify-users).

**Comment by breyed** [comment]
For now, it would be good to focus on ensuring the docs prevent further confusion. It's probably just a matter of ensuring the XML comments are complete and getting them into the NuGet package (#818).

**Comment by breyed** [comment]
Good point about the "Identify users" docs. Unfortunately, they're not easily discoverable if you happened to notice `SetUserId` in the process of coding.

While the docs there say that the user ID is used in crash reporting, they don't say that it's not used elsewhere. On one hand, since only crash reporting is mentioned, one could assume that's all it's used for. On the other hand, the API isn't in the `Crashes` namespace, which you might expect for something specific to crashes. So it's probably best in this case to be explicit.

**Comment by blparr** [comment]
Thank you - that's definitely a fair point. We will do a docs update incorporating your feedback.
